sopel/tools/__init__.py

```diff
@@ -620,24 +501,51 @@ class SopelIdentifierMemory(SopelMemory):
     with both ``Identifier`` and :class:`str` objects, taking advantage of the
     case-insensitive behavior of ``Identifier``.
 
+    As it works with :class:`~.identifiers.Identifier`, it accepts an
+    identifier factory. This factory usually comes from a
+    :class:`bot instance<sopel.bot.Sopel>`, like in the example of a plugin
```

Comment:
Low priority: ``:class:`bot instance<sopel.bot.Sopel>`,`` will render "`bot instance`" in fixed-width font (see [deploy preview](https://deploy-preview-2231--sopel-unstable-docs.netlify.app/api.html#sopel.tools.SopelIdentifierMemory)). This is not _broken_ per se, but it's one recurring annoyance that I have with Sphinx docs. Sometimes you want to link a class or function with custom, normal text. 😤

Something like this could work?

```suggestion
    :ref:`bot instance<py:class:sopel.bot.Sopel>`, like in the example of a plugin
```

Reply:
Oh... you are right. I'll try that, otherwise I think I know a way.